### DISCUSSION TITLE
Is it possible to forward messages from channel to bot?

### DISCUSSION THREAD

**Discussion by manojsitapara** [aahnik/tgcf#188] [Q&A] [answered]
I have following code on my tgcf.config.yml file

Source is - channel 
Destination is - Bot

![image](https://user-images.githubusercontent.com/12126671/120516410-f381cb80-c3ec-11eb-9611-6e05165edfc6.png)

It gives following error

`ValueError: Could not find the input entity for PeerUser(user_id=1746238586) (PeerUser). Please read https://docs.telethon.dev/en/latest/concepts/entities.html to find out more details.`

BUT

If I use source and destination both as channel, it is working fine without any issue, only when I set destination as bot.

NOTE: I am using user account login.

Is it not possible to forward message from channel to bot or I am doing something wrong?

**Comment by ankit-sinha-18** [comment, marked as answer]
Put Bot username in destination chat
It will definitely work

`dest : [telegrambot]`

**Reply by manojsitapara** [reply]
Thank you !! Yes it worked without @ as follow

`dest : [Mybot_bot]`

**Reply by ankit-sinha-18** [reply]
Yes, it was my mistake
